### PR TITLE
User/vchintalapat/GitHub actions test

### PR DESCRIPTION
**Pull request by Saipriya-1144**
--> Added a github_actions_test workflow and a sample .NET app to this repository 
--> The github_actions_test workflow runs every 3 hours, updates a placeholder in the dotnet and node app with the current UTC time, and deploys the app to Azure App Service.

### CODE REVIEW

**Comment by sarsharma** [comment]
let's add a test for nodejs too, especially for scenario where app is prebuilt by cx

**Comment by ShilpiRachna1** [comment]
Looks good!

File: .github/workflows/github_actions_test.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          client-id: ${{ secrets.AZUREAPPSERVICE_CLIENTID_NODEAPP1 }}
+          tenant-id: ${{ secrets.AZUREAPPSERVICE_TENANTID }}
+          subscription-id: ${{ secrets.AZUREAPPSERVICE_SUBSCRIPTIONID }}
+      
+      - name: 'Deploy to Azure Web App'
+        id: deploy-to-webapp
+        uses: azure/webapps-deploy@v3
+        with:
+          app-name: 'lwasv2-euap-node-githubactions'
+          slot-name: 'Production'
```
Comment (ShilpiRachna1) [inline .github/workflows/github_actions_test.yml:110]: Can be a separate PR. Can we change one of these deployments to deploy to a non-production slot

**Review by ShilpiRachna1 — APPROVED** [review]
Looks good! Left a non-blocking comment

**Review by kumaraksh1 — APPROVED** [review]
LGTM
